npm/create-cypress-tests/src/component-testing/babel/babelTransform.ts

```diff
@@ -80,7 +88,24 @@ export function createTransformPluginsFileBabelPlugin (ast: PluginsConfigAst): b
             path.parent.right.params.push(babelTypes.identifier('config'))
           }
 
-          path.get('body').pushContainer('body' as never, ast.ModuleExportsBody)
+          const statementToInject = Array.isArray(ast.IfComponentTestingPluginsAst)
+            ? ast.IfComponentTestingPluginsAst
+            : [ast.IfComponentTestingPluginsAst]
+
+          const ifComponentMode = babelTypes.ifStatement(
+            babelTypes.binaryExpression(
+              '===',
+              babelTypes.identifier('config.mode'),
```

Comment:
@lmiller1990 once you will came up with the final API – please edit this line 
```suggestion
              babelTypes.identifier('config.testingType'),
```

And update snapshots to verify everything works as expected

Reply:
Ok, I can do it.